.github/workflows/Release.yml

```
@@ -0,0 +1,82 @@
+name: "Publish Release"
+on:
+  workflow_dispatch:
+env:
+  build_dir: ./build
+  tarballs_dir: ./build/tarballs
+jobs:
+  release:
+    name: "Publish Pre-Release"
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          fetch-depth: 0  # Needed determine latest tag
```

As you are relying on this step to persist credentials we may as well be explicit about it:
```suggestion
          fetch-depth: 0  # Needed determine latest tag
          persist-credentials: true
```